source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+/* Base Styles */
+body {
+    scrollbar-gutter: stable;
+    cursor: default !important;
+  }
+  
+  body.stop-scrolling {
+    height: 70%;
+    overflow: inherit;  /* disable SweetAlert killing the scroll bar (stops the wiggle) */
+  }
+  
+  .body_sidebarScroll {
+    overflow: hidden !important;
+  }
+  
+  /* Typography */
+  a {
+    color: var(--a-color);
+  }
+  
+  .ca_bold {
+    font-weight: bold;
+  }
+  
+  .ca_italic {
+    font-style: italic;
+  }
+  
+  .ca_normal {
+    font-size: 1.4rem !important;
+  }
+  
+  .ca_note {
+    color: #d67777;
+  }
+  
+  .ca_red {
+    color: #882626;
+  }
+  
+  .ca_green {
+    color: #038503;
+  }
```

Comment:
trying using values from the [webgui's default-color-palette.css](https://github.com/unraid/webgui/blob/3cfa60e726b4a9df8ab581dcc1174f966211d114/emhttp/plugins/dynamix/styles/default-color-palette.css)